sirepo/template/srw.py

```diff
@@ -1286,6 +1254,13 @@ def _enum_text(name, model, field):
     return ''
 
 
+def _export_rsopt_config(data):
+    r = 'exportRsOpt'
+    v = _rsopt_jinja_context(r, data.models[r])
+    v.libFiles = _SIM_DATA.lib_file_basenames(data)
```

Comment:
Simplify:

```py
return _write_rsopt_zip(data, _rsopt_jinja_context(data))
```

`_rsopt_jinja_context` can know the constant `exportRsOpt`, which actually could be a constant in the module and used in _OUTPUT_FOR_MODEL and write_parameters.